dadb/src/main/kotlin/dadb/AdbShell.kt

```diff
@@ -93,27 +104,19 @@ class AdbShellStream(
     }
 }
 
-class AdbShellPacket(
-        val id: Int,
-        val payload: ByteArray
+sealed class AdbShellPacket(
```

Comment:
Let's keep the `id` field for a couple reasons:

* It makes this a non-breaking change
* It's necessary for Java code to consume this object in a sane manner (instead of checking classes)